src/fprime_gds/executables/cli.py

```diff
@@ -340,6 +340,8 @@ def handle_arguments(self, args, **kwargs):
                     args.logs, datetime.datetime.now().strftime("%Y_%m_%d-%H_%M_%S")
                 )
             )
+            # A dated directory has been set, all log handling must now be direct
+            args.log_directly = True
```

Comment:
I noticed there's a `first_log` class variable that seems to have a similar purpose, although it's not used. Could be worth it to remove it if there's no plan to use it in the future?
https://github.com/fprime-community/fprime-gds/blob/4ec5fa1403d33dbde0537e5dc8b42cedfacf07d8/src/fprime_gds/executables/cli.py#L300-L302

Reply:
Yes we should remove it!  I'll do so on this PR as we'll not remember to do so later.